doc/taoQtiTestPreviewer.yaml

```diff
@@ -0,0 +1,247 @@
+openapi: 3.0.1
+info:
+  title: taoTestPreview API
+  description: Stands for TAO test Previewer API.
+  version: 1.0.0
+servers:
+  - url: '{server}'
+    variables:
+      server:
+        default: https://community.docker.localhost/
+security:
+  - cookieAuth: []
+paths:
+  /taoQtiTestPreviewer/TestPreviewer/init:
+    get:
+      parameters:
+        - in: query
+          name: testUri
+          schema:
+            type: string
+          required: true
+          example: 'https%3A%2F%2Fcommunity.docker.localhost%2Fontologies%2Ftao.rdf%23i5efb42fced1ce7064796c08b38ee176'
```

Comment:
localhost?

Reply:
Yes, all places are using localhost. What do you suggest?